packages/jest-jasmine2/src/jasmineAsyncInstall.ts

```
@@ -223,6 +223,8 @@ function makeConcurrent(
   };
   // each is binded after the function is made concurrent, so for now it is made noop
   concurrentFn.each = () => () => {};
+  // "failing" is not implemented in jasmine. It has to be defined because of TS types.
+  concurrentFn.failing = () => () => {};
```

we should throw an error instead of no-oping